plugins/catalog-backend-module-ldap/package.json

```diff
@@ -20,31 +23,43 @@
     "backstage"
   ],
   "scripts": {
+    "postbuild": "tsc -p tsconfig.cli.json",
     "build": "backstage-cli backend:build",
     "lint": "backstage-cli lint",
     "test": "backstage-cli test",
     "prepack": "backstage-cli prepack",
     "postpack": "backstage-cli postpack",
-    "clean": "backstage-cli clean"
+    "clean": "backstage-cli clean",
+    "dev": "nodemon --watch \"src/**\" --ext \"ts\" --ignore \"*.spec.ts\" --exec \"ts-node --project tsconfig.cli.json src/cli/index.ts\""
```

Comment:
Please stick to the existing ways of running standalone node packages, see for example `codemods` or `cli`